.docker/entrypoint/spec/jekyll_server_spec.rb

```diff
@@ -49,6 +49,10 @@
       it {
         expect(File).not_to exist(File.join(__dir__, '..', '0.0.0.0'))
       }
+
+      it {
+        expect(Dir.empty?(File.join(data_dir, "_site"))).not_to be_empty
```

Comment:
Codacy found an issue: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5820027)